components/Header.js

```diff
@@ -0,0 +1,65 @@
+import Image from "next/image";
+// if we wanted to render home page header these data will be passed pic = "/../public/home.png", buttons = "true",
+// if we wanted to render roadmaps header these data will be passed pic = "/../public/Roadmaps.png", light = "true", text = "true"
+
+export default function Header({
+  text = true,
+  buttons,
+  light = true,
+  pic = "/../public/Roadmaps.png",
+}) {
+  return (
+    <div
+      className="grid grid-cols-2 gap-5 text-white   pl-6 pr-6 pt-6 pb-6    lg:pl-16 lg:pr-16 lg:pt-16 lg:pb-16   md:pl-12 md:pr-12 md:pt-12 md:pb-12"
```

Comment:
instead of `pl-6 pr-6 pt-6 pb-6 lg:pl-16 lg:pr-16 lg:pt-16 lg:pb-16` you can do `p-6 lg:p-16`

Reply:
sorry I'm a bit new to tailwind, will do more research before doing something next time